bin/mag.js

```diff
@@ -0,0 +1,104 @@
+    query_string_before= "OR("
+    query_string_after=")"
+    words = str.toLowerCase().split(' ')
+    const filteredArray = words.filter(function(x) { 
+        return stopwords.indexOf(x) < 0;
+      });
+    filteredArray.forEach(function (item,index, array){
+        if (index === (array.length -1)) {
+            // This is the last one.
+            query_string_before +="W='"+item+"'"
+        }else{
+            query_string_before +="W='"+item+"',"
+        }
+    });
+    
+    return(query_string_before+query_string_after+"")
+}
+
+
+
+function handle_project(app, cb) {
```

Comment:
It should be named "project"? Please use camelCase for all variable / function names.